src/main/java/io/supertokens/inmemorydb/queries/TOTPQueries.java

```diff
@@ -262,6 +249,7 @@ public TOTPUsedCode map(ResultSet result) throws SQLException {
                     result.getBoolean("is_valid"),
                     result.getLong("expiry_time_ms"),
                     result.getLong("created_time_ms"));
+            // FIXME: Put created time first, then expiry time.
```

Comment:
FIXME - not sure why you moved this PR into review with a FIXME

Reply:
It isn't required. But it would be better to do this. It should have been a TODO.